app/controllers/users_controller.rb

```diff
@@ -8,4 +8,34 @@ def show
     @user = current_user
   end
 
+  def edit
+    @user = current_user
+  end
+
+  def update
+    @user = current_user
+    # avatar = params[:user][:avatar]
```

Comment:
Take out any hanging comments like this. It's messy code that might confuse you/ your team in the future